packages/react/src/components/button/button.tsx

```diff
@@ -10,7 +10,7 @@ export const Button: ButtonComponent = ({
 }: ButtonProps<'a'> | ButtonProps<'button'>) => {
   const { disabled } = useField();
 
-  const Element = 'href' in restProps ? 'a' : 'button';
+  const Element = (restProps as ButtonProps<'a'>).href ? 'a' : 'button';
```

Comment:
What do you think about having this change as a separate PR as a "react fix"?

Might be good to have such in a changelog?

Reply:
Hmm it shouldn't impact TypeScript users, but might JavaScript ones.

Yea I'll split it 👍🏻

Reply:
https://github.com/onfido/castor/pull/576